web/components/notification-settings.tsx

```diff
@@ -477,3 +450,100 @@ export function NotificationSettings(props: {
     </div>
   )
 }
+
+export const notificationIsNecessary = (
```

Comment:
Why the move and export if it's still only used in this file?

Reply:
I want to use it in the future for setting preferences from the notifications tab